NorthstarDLL/squirrel.h

```diff
@@ -77,11 +78,32 @@ struct SQFuncRegistration
 
 enum class ScriptContext : int
```

Comment:
these shouldn't be bitflags, this enum should mirror the context ids that respawn use (i.e. sv = 0, cl = 1, ui = 2)